lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -19,9 +19,14 @@
             <% } %>
         </span>
         <span class="sr timer-announce" aria-live="assertive"></span>
-        <span id="time_remaining_id">
-            <b>
-            </b>
+        <span class="exam-timer-clock">
+            <span id="time_remaining_id">
+                <b>
+                </b>
+            </span>
+            <button id="toggle_timer" title="<%- gettext("Hide Timer") %>" aria-label="<%- gettext("Hide Timer") %>">
```

Comment:
@jspayd I just did some implementation testing and got some unfortunate results.  The accessible naming algorithm for buttons is unlike any other element. It's the one element where the title attribute value contributes to the accessible name.  So when you use both the title attribute and the aria-label, both are used to determine the accessible name for the control in all screen readers I tested (VO, JAWS, NVDA)...  So as you can imagine, it got pretty confusing when the button was toggled to hide the timer.  It said some variation of "Hide timer toggle button pressed Show timer" and then "Hide button toggle button not pressed Hide timer."

You have a couple options here:

1. get rid of the title attribute all together and just toggle the aria-pressed attr. This results in the best UX for the SR user, but there is no title attribute to help sighted mouse users.
2. Since you are changing the value of the accessible name, the title and the icon (wasn't aware you were already doing this), you can leave it as it is.  The only problem is the SR user has to leave the control, then go back to it to confirm the intended action took place. It is not read to them as part of the action.